container/build.gradle

```diff
@@ -7,7 +7,10 @@ dependencies {
 
     api project(":model")
 
-    api "org.apache.groovy:groovy-all:$groovyVersion"
+    api "org.apache.groovy:groovy:$groovyVersion"
+    api "org.apache.groovy:groovy-json:$groovyVersion"
+    api "org.apache.groovy:groovy-xml:$groovyVersion"
+    api "org.apache.groovy:groovy-yaml:$groovyVersion"
```

Comment:
When [Grape dependency managment](https://docs.groovy-lang.org/latest/html/documentation/grape.html) (`@Grab` annotation) should be supported, we also need to add the Ivy dependencies.